blank-canvas-blocks/sass/blocks/_button.scss

```diff
@@ -7,7 +7,8 @@
 	font-family: var(--wp--custom--button--font-family);
 	font-size: var(--wp--custom--button--font-size);
 	line-height: var(--wp--custom--button--line-height);
-	border-radius: var(--wp--custom--button--border-radius);
+	border-radius: var(--wp--custom--button--border--radius);
+	border-width: var(--wp--custom--button--border--width);
```

Comment:
Because of the unique calculations that the button block is doing with the assumption of a 2px border-width (to make the regular and outline-style buttons the exact same height) I'm a little nervous opening that up for configuration.